deploy/examples/multus-validation.yaml

```diff
@@ -1,11 +1,44 @@
-#################################################################################################################
-# We highly recommend validating your Multus configuration before you install Rook.
-# This job aims to automate that operation by using the validation tool. Run this job after
-# installing the rook operator and before installing any Custom Resources.
-# Insert the NAD name for public network and cluster network in the Job definition below.
-# If you want to use any other flags along with the basic command in the Job,
-# add the `--help` flag in the end to see the list of flags available, and use accordingly.
-#################################################################################################################
+####################################################################################################
+# This manifest contains a Kubernetes Job and supporting definitions for running Rook's Multus
```

Comment:
notice the ci failure for yaml-linter

Reply:
Thanks. fixed by fixing the script and committing the updated result

Reply:
@subhamkrai  thanks for the suggestion!

if I get it right, the main point here is to add a copyright/license comment header.

If this is required by the rook project, I will certainly abide, but as far as I know, this practice is generally considered unnecessary and redundant in an open source project that has a license and copyright information on the entire project. @travisn / @BlaineEXE / @galexrt   - what is your take on this?